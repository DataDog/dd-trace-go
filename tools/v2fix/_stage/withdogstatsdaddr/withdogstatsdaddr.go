// Unless explicitly stated otherwise all files in this repository are licensed
// under the Apache License Version 2.0.
// This product includes software developed at Datadog (https://www.datadoghq.com/).
// Copyright 2024 Datadog, Inc.

package main

import (
	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
)

func main() {
	tracer.Start(tracer.WithDogstatsdAddress("localhost:8125")) // want `the function WithDogstatsdAddress is no longer supported. Use WithDogstatsdAddr instead.`
	tracer.Stop()
}
