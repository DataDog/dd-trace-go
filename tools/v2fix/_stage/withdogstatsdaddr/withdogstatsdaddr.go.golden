-- the function WithDogstatsdAddress is no longer supported. Use WithDogstatsdAddr instead --
// Unless explicitly stated otherwise all files in this repository are licensed
// under the Apache License Version 2.0.
// This product includes software developed at Datadog (https://www.datadoghq.com/).
// Copyright 2024 Datadog, Inc.

package main

import (
	ddtrace "gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
)

func main() {
	ddtrace.Start(ddtrace.WithDogstatsdAddr("localhost:8125")) // want `the function WithDogstatsdAddress is no longer supported. Use WithDogstatsdAddr instead`
	ddtrace.Stop()
}
